tests/main/user-mounts/task.yaml

```diff
@@ -9,45 +9,35 @@ details: |
 # This test makes use of the user mount provided by the desktop
 # interface.  When we have an interface providing user mounts that is
 # available on core, we can switch to that.
-systems: [-ubuntu-core-*]
+systems:
+    - -ubuntu-core-*   # see above
+    - -ubuntu-14.04-*  # no session-tool
```

Comment:
Hmm, not sure if we want to drop support for 14.04 if the test was previously passing as that suggests that session-tool isn't testing the same thing. 

Is it the case that this test previously on 14.04 wasn't really testing anything? If so this is probably fine.

Alternatively, are we okay with decreasing test coverage of 14.04 desktop things? Probably unlikely that ESM is used for desktops and probably more for servers I would think, but I really don't know...

Reply:
Added a TODO in https://github.com/snapcore/snapd/pull/8603